### ISSUE TITLE
Add Rank as well as Platform as an LFG

### ISSUE THREAD

**Issue by Starga123** [malouro/ggis-bot#4]
Hey mate,

Not sure if you check this anymore as I see that the last time it was updated was 10 months ago, but I've been trying my best to add Platform and Rank to the currently LFG code, could you shed some light on how I can approach this?

TIA

**Comment by malouro** [comment]
Hey man, would you mind elaborating on the question a bit further? I'm assuming by platform you mean like what console the game is from right? ie: PC, PS4, Xbox, etc.? What about 'rank'?

If I'm right about what you mean with platform, then there's a couple ways to go about it, but I'd rather get the full picture of what you want to accomplish before diving down that rabbit hole 😛

**Comment by malouro** [comment]
And thanks for checking out the project! Lemme know how it's been working for you.

It's not very maintained and out of date, and since it was one of the first things I've done in JavaScript, there's plenty of cruft in the code. Curious to see if you've had any issues getting a bot up and running, following any of the documentation, or navigating around the structure of the project/code.

**Comment by Starga123** [comment]
Hey man,

Sorry for the poor description!
Yes exactly that, so when a member searches for example !lfg rl rankedduo they would also need to add their platform and rank. By rank I mean Grand Champ or Diamond for example :)

I've had a try myself to add these under the modes_proper (but changed it to rank + platform) but can't get my head around it at all.

I've been searching for a LFG bot for a while and yours is definitely the cleanest looking one I've managed to find and it's actually helped me learn Java myself as I'm a complete novice. I did actually manage to get it up and running on a test server and it works without any issues, I'm actually planning to add it to a server that I'm a manager of with 6000 members. Will definitely make sure I give you credit within a command too. The documentation and structure etc was very easy to follow, as I mentioned I'm a complete Novice and managed to figure everything out (even it did take my roughly 3-4 days to finish everything off)

If you want, feel free to add my on Discord as it'll probably be easier to communicate there!

Starga#3436
